dlang.org.ddoc

```diff
@@ -436,12 +435,14 @@ SUBNAV_HEAD=
 SUBNAV_TEMPLATE=$(DIVC subnav-helper) $(DIVC subnav, $0)
 _=
 
-SUBNAV_TOOLS=
+SUBNAV_CLI_REFERENCE=
 $(SUBNAV_TEMPLATE
-    $(SUBNAV_HEAD D-Specific Tools, $(ROOT_DIR)tools.html, overview)
+    $(SUBNAV_HEAD Command-line Reference, http://wiki.dlang.org/Development_tools, overview)
```

Comment:
Wiki page is less of an overview and more of a "see more" thing. Perhaps it could be moved at the bottom as a "more tools" link OSLT?